config/00-main.yaml

```diff
@@ -23,6 +23,13 @@ score_caches:
     prefix: ores
     socket_timeout: 15 # seconds
 
+#Lock managers
```

Comment:
Doesn't seem like this should be the default, since most small installations will probably have a single client, and might intentionally want to allow max thoroughput.

Reply:
Yes, I think we should remove it from defaults in below